Packages/com.unity.inputsystem/InputSystem/Devices/Keyboard.cs

```diff
@@ -1670,8 +1737,8 @@ public string keyboardLayout
         /// The plus ('+') key on the numpad. The key on the right side of the numpad with
         /// <see cref="numpadMinusKey"/> above it and <see cref="numpadEnterKey"/> below it.
         /// </summary>
-        /// <value>Control representing the numpad plus key.</value>
         /// <remarks>
+        /// <see cref="KeyControl"/> representing the numpad plus key.
```

Comment:
Needs a paragraph after it?

Reply:
fixed in [8ad97b6](https://github.com/Unity-Technologies/InputSystem/pull/2069/commits/8ad97b6e1a384f347459dbfa06ba72c998464ce4)